### ISSUE TITLE
Handle RecordNotUnique: app/models/organizations/judge_team.rb in add_user

### ISSUE THREAD

**Issue by yoomlam** [department-of-veterans-affairs/caseflow#13050]
## Description

Handle `ERROR:  duplicate key value violates unique constraint`

Might just be user error, but we've recently changed code here.

## Background/context/resources

During Bat Teaming, https://dsva.slack.com/archives/C4JECDLSE/p1576712970056100.

Sentry alert: https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/7608/

## Technical notes
`duplicate key value violates unique constraint "index_judge_team_roles_on_organizations_user_id" DETAIL: Key (organizations_user_id)=(4898) already exists. : INSERT INTO "judge_team_roles" ("type", "organizations_user_id", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id"`

```ERROR:  duplicate key value violates unique constraint "index_judge_team_roles_on_organizations_user_id"
DETAIL:  Key (organizations_user_id)=(4898) already exists.```

app/models/organizations/judge_team.rb in block in add_user at line 37:
```ruby
  def add_user(user)
    super.tap do |org_user|
      class_name = (users.count == 1) ? JudgeTeamLead : DecisionDraftingAttorney
      class_name.create!(organizations_user: org_user)
    end
  end
```

**Comment by hschallhorn** [comment]
May be fixed in #13191

**Comment by araposo-tistatech** [comment]
@hschallhorn did this one end up getting fixed in #13191?

**Comment by hschallhorn** [comment]
Yes!
